src/components/Quiz/AnswerButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
+
+interface AnswerButtonProps {
+  children: ReactNode;
+  onClick: () => void;
+}
+
+export default function AnswerButton({ children, onClick }: AnswerButtonProps) {
```

Comment:
We use arrow functions in this project

Reply:
We should add the ESLint rule to keep the consistency